CMakeLists.txt

```diff
@@ -1,27 +1,5 @@
 cmake_minimum_required(VERSION 3.21)
```

Comment:
We aren't having a top-level CMakeLists.txt file, so you can delete this. To build examples, you run cmake from the examples directory.